versions/development/SPEC.md

```diff
@@ -1049,6 +1057,43 @@ output {
 }
 ```
 
+#### Directory Outputs
+
+Directories can be declared as outputs just like files. For example:
+```
+output {
+  Directory d = "created/directory/"
+}
+```
+
+Notes:
+
+ - When a directory is output, the contents (including subdirectories) are considered part of the output but the path is not. 
+ - When a directory is output, its contents *do not overwrite* any pre-existing directory content. For example:
+
+```wdl
+workflow directory_manipulation {
+  # Make a directory:
+  call make_directory
+  Directory d = make_directory.made_directory
+  
+  # List the contents before the "update_directory" task:
+  call list_contents as list_contents_before { input: d = d }
+  String d_contents_before = list_contents_before.list
+
+  # An update task:
+  call update_directory { input: d = d }
+  
+  # List the contents again:
+  call list_contents as list_contents_after { input: d = d }
```

Comment:
Actually this example isn't great - since this `call` doesn't depend on any outputs of `update_directory`, it would all just run in parallel 🤦‍♂️